### ISSUE TITLE
Update the tutorial Creating a basic machine learning system on Open MLOps with sample values from local minikube

### ISSUE THREAD

**Issue by vfortu** [datarevenue-berlin/OpenMLOps#92]
it would be great to have also the default values of url and domains when setting up in a local environment with the defaults  my_vars.tfvars defined in the minikube guide. So for example:

how these
      aws = false
      db_username = "mlflow-db-user"
      db_password = "mlflow-db-pasword"
      hostname = "myambassador.com"
      ory_kratos_cookie_secret = "secret"
      ory_kratos_db_password = "password"
      install_metrics_server = false
      install_feast = false
      install_seldon = false
      prefect_create_tenant_enabled = false
      jhub_proxy_secret_token = "IfYouDecideToUseJhubProxyYouShouldChangeThisValueToARandomString"
      enable_ory_authentication = false
      oauth2_providers = []
      mlflow_artifact_root = "/tmp"

match with
    domain = "mlops.ritza-route53.com"  # the domain where you are hosting Open MLOps
    username = "gareth@ritza.co"        # the username you used to register on Open MLOps
    password = "DkguP5GsB9yiPk8"        # the password you used to register on Open MLOps
    s3_bucket = "another-mlops-bucket"  # the S3 bucket you specified when setting up Open MLOps
    
    prefect_project_name = "wine-quality-project"         # you can use what you want here
    docker_image = "drtools/prefect:wine-classifier-3"    # any docker image that has the required Python dependencies

or also 
 how to reach  in the minikube case:
      =>  https://mlops.example.com/profile/auth/registration  
       => https://jupyter.mlops.example.com

that would be great thanks!

**Comment by pipatth** [comment]
Hi @vfortu,

Sorry for the bug. This is fixed in https://github.com/datarevenue-berlin/OpenMLOps/pull/99

Please be aware that the instructions are a little different (different myvars.tfvars, port-forward). Let me know if this fix works for you. Thank you.

**Comment by sramirez** [comment]
Hi all,

Actually, if you install the stack without ory authentication things get more complicated when configuring prefect connection. I tried with different configurations but no success till now. Would it possible to replace the ipynb file with a more basic example around Prefect?

**Comment by bernardolk** [comment]
> Hi all,
> 
> Actually, if you install the stack without ory authentication things get more complicated when configuring prefect connection. I tried with different configurations but no success till now. Would it possible to replace the ipynb file with a more basic example about the Prefect part?

Hello. If I understand correctly you are suggesting that we don't use Ory in the minikube tutorial/setup, right?

**Comment by sramirez** [comment]
Yes, maybe in this way the minikube tutorial will be closer to a real scenario with local deployment w/o authentication.

I tried to connect with the following sentences:

```
prefect_url = f"http://IP:4200/graphql"
prefect_client = Client(api_server=prefect_url)
```
I managed to launch an scheduled job in Prefect, but at the same time getting errors with the command. On the other hand, it was not possible the creation of projects from python (I did it manually from the UI).
